cmd/openshift-tests/e2e.go

```diff
@@ -85,6 +85,7 @@ var staticSuites = testSuites{
 				}
 				return strings.Contains(name, "[Suite:openshift/conformance/serial") || isStandardEarlyOrLateTest(name)
 			},
+			TestTimeout:         20 * time.Minute,
```

Comment:
I'm fine bringing back the extra wait, but I don't want to increase the overall test timeout for everyone. Spoke with @bfournie he'll try to make it more targetted at bare metal.